assets/src/edit-story/panels/size.js

```diff
@@ -28,12 +30,9 @@ function SizePanel( { selectedElements, onSetProperties } ) {
 		evt.preventDefault();
 	};
 	return (
-		<Panel onSubmit={ handleSubmit }>
-			<Title>
-				{ __( 'Size', 'amp' ) }
-			</Title>
-			<InputGroup
-				label={ __( 'Width', 'amp' ) }
+		<RowPanel onSubmit={ handleSubmit }>
+			<PrefixInput
+				label={ __( 'W', 'amp' ) }
 				value={ state.width }
 				isMultiple={ width === '' }
 				onChange={ ( value ) => {
```

Comment:
Please optimize these callbacks by e.g. moving them to a memoized `useCallback` above. And maybe use the same callback for both inputs, as they're kind of similar but not completely? If possible?.

Reply:
handleSubmit is on useCallback now.

Reply:
For those two values, width and height, we need to indicate which is being modified.
Currently I implemented two separate handles. I think to merge them, we can use new state with onFocus on prefixInput.

Reply:
Hm, I don't understand this concept? How would you use `onFocus` to merge this to a single handler?

I was thinking something with a curried function along the lines of:

```js
const handleChange = useCallback( ( prop ) => ( evt ) => { ... }, [] );
...
<PrefixInput ... onChange={ handleChange( 'width' ) } />
<PrefixInput ... onChange={ handleChange( 'height' ) } />
```

But it might not actually improve the code, just make it more messy. However, if there is shared functionality, please try to generalize if possible.